test_unstructured/partition/pdf_image/test_pdf.py

```diff
@@ -208,48 +208,34 @@ def test_partition_pdf_local_raises_with_no_filename():
 
 @pytest.mark.parametrize("file_mode", ["filename", "rb", "spool"])
 @pytest.mark.parametrize(
-    ("strategy", "starting_page_number", "expected_page_numbers", "origin"),
+    "strategy",
     # fast: can't capture the "intentionally left blank page" page
     # others: will ignore the actual blank page
     [
-        (PartitionStrategy.FAST, 1, {1, 4}, {"pdfminer"}),
-        (PartitionStrategy.FAST, 3, {3, 6}, {"pdfminer"}),
-        (PartitionStrategy.HI_RES, 4, {4, 6, 7}, {"yolox", "pdfminer", "ocr_tesseract"}),
-        (PartitionStrategy.OCR_ONLY, 1, {1, 3, 4}, {"ocr_tesseract"}),
+        PartitionStrategy.FAST,
+        PartitionStrategy.HI_RES,
+        PartitionStrategy.OCR_ONLY,
     ],
 )
 def test_partition_pdf_outputs_valid_amount_of_elements_and_metadata_values(
```

Comment:
Can you explain why you made changes to this test? I don't believe it's related to your PR

Reply:
@Coniferish 
It's because, test_partition_pdf_with_password() is a copy of test_partition_pdf_outputs_valid_amount_of_elements_and_metadata_values(), and you ask me to simplify this tests.
I can re-duplicate the test_partition_pdf_outputs_valid_amount_of_elements_and_metadata_values() in test_partition_pdf_with_password().
what do you think?

Reply:
@Coniferish you can relaunch the CI/CD.